packages/dds/tree/.eslintrc.cjs

```diff
@@ -22,7 +22,7 @@ module.exports = {
 				caughtErrorsIgnorePattern: "^_",
 			},
 		],
-		"import/no-internal-modules": "off",
+		"import/no-internal-modules": ["error", { allow: ["**/index.js"] }],
```

Comment:
I think this is closer to what we want:
```suggestion
		"import/no-internal-modules": ["error", { allow: ["**/../*/index.js", "./*/index.js"] }],
```

Reply:
ideally the "**/../*/index.js" case would not allow ".." for that single * entry, since importing parent index files is also bad (tends to make dependency cycles), but I'm not sure of a good way to bad that and we have other ways to catch it

Reply:
It looks like `no-internal-modules` resolves `..` before glob match. So far exactly allowing `*/index.js` will work.

Reply:
Parents are allowed with the revised change, but I think that is the best we can do.